src/main/resources/modules/allergies.json

```diff
@@ -158,15 +158,15 @@
       "codes": [
         {
           "system": "SNOMED-CT",
-          "code": "384758001",
+          "code": 384758001,
```

Comment:
I didn't notice this before but there are some instances in the diff like this one, where neither the code nor display value changed, but the code was changed from a string to a number. Any idea what happened here? It doesn't seem like all codes were changed. This isn't a huge deal but if it's possible to not change the type if there wasn't a value change, or just make them all strings instead of numbers, that would be preferable to making them numbers